client/components/data/query-domain-owner-username/index.ts

```diff
@@ -47,7 +47,7 @@ export function useDomainOwnerUserName(
 	//Due to Jetpack sites overriding the user.ID with a completely different thing,
 	//when Jetpack overrides this property, the original WordPress.com user Id
 	//ends stored as user.linked_user_ID, so in those cases, that's the ID we have to use.
-	const ownerUser = teams.users?.find(
+	const ownerUser = teams?.users?.find(
```

Comment:
Looks like we have potencially the same problems with `email`, so I would fix it too, at least to have consistency [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/data/query-email-owner-username/index.ts#L47)

Reply:
Also, it would be perfect to avoid type assertion and get the type of data automatically from the hook. It would prevent the future similar problems. It's not critical and just nice to have, so up to you.